.vscode/settings.json

```diff
@@ -1,10 +1,13 @@
 {
   "editor.defaultFormatter": "esbenp.prettier-vscode",
   "editor.formatOnSave": true,
-  "editor.rulers": [120],
+  "editor.rulers": [
```

Comment:
nit: is this intended?

Reply:
I got not difference

```
❯ prettier --version
3.2.4

❯ prettier --list-different  ./tsconfig.json
tsconfig.json
```
What version are you using?

Reply:
looks like this is another hickup with setup/CI. I fixed in #18
Apologies for that :) the code hasn't been updated in a few years. Rebasing should make this go away.